src/components/Competitiveness.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Text} from "react-native";
+import { Box, Stack } from "@mui/material";
+
+function Competitiveness(competition) {
+  return(
+    <Stack direction="row">
+      <Box
+        sx={{
+          backgroundColor: () => {
+            let comp = competition.toLowerCase();
+            if (comp == "ultra competitive") return "red";
+            else if (comp == "very competitive") return "#ECE54B";
+            else if (comp == "competitive") return "#48C246";
+            else return "blue";
```

Comment:
I changed this in the uni search page to return transparent, because there are some cases where passport doesnt have the competitiveness, so we shouldnt display anything